figures/scripts/fig4-tp53-telomerase-panels.R

```diff
@@ -416,28 +422,64 @@ dev.off()
 
 
 ## Survival analysis forest plot -----------------------------------------------------------------
+resect_ref <- "Tumor resection: Biopsy (ref)"
+lgg_ref <- "LGG group: non-LGG (ref)"
+hgg_ref <- "HGG group: non-HGG (ref)"
+
+# Set up ordering and labels for y-axis
+term_order <- rev(c("tp53_score",
+                    "telomerase_score",
+                    "extent_of_tumor_resectionGross/Near total resection",
+                    "extent_of_tumor_resectionPartial resection",
+                    resect_ref,
+                    "lgg_groupLGG",
+                    lgg_ref,
+                    "hgg_groupHGG",
+                    hgg_ref))
+
+term_labels <- rev(c("TP53 score",
+                     "Telomerase score",
+                     "Tumor resection: Total",
+                     "Tumor resection: Partial",
+                     resect_ref,
+                     "LGG group: LGG",
+                     lgg_ref,
+                     "HGG group: HGG",
+                     hgg_ref))
 
-survival_dir <- file.path(analyses_dir, "survival-analysis", "results")
-survival_result <- readRDS(file.path(
-  survival_dir,
-  "cox_additive_terms_tp53_telomerase_hgg.RDS"
-))
 
 # Get n and event info from glance outpout
 survival_n <- broom::glance(survival_result) %>%
   select(n, nevent)
 
 # Convert survival model result to data frame, and exponentiate estimates/CIs to get HRs
 survival_df <- broom::tidy(survival_result) %>%
+  # add references
+  add_row(term = resect_ref, estimate = 0) %>%
+  add_row(term = lgg_ref, estimate = 0) %>%
+  add_row(term = hgg_ref, estimate = 0) %>%
+  #remove unknown resection from plot
+  filter(term != "extent_of_tumor_resectionUnavailable") %>%
```

Comment:
Why is this being removed from the plot? It's definitely part of the model [here](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/survival-analysis/survival-analysis_tp53_telomerase.nb.html#additive-model-tp53_scoretelomerase_scoreextent_of_tumor_resectionlgg_grouphgg_group).

Reply:
Noting we discussed that this term should actually not be in the model, so this PR has been filed to take it out of the model, which will facilitate this plot soon - https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1356

Reply:
@sjspielman the FP has been rerun with #1356 in